recipes/dnslib/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "dnslib" %}
+{% set version = "0.9.10" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 666bf55274a630a2925411c4ea9ca724529299bbe70f91979ad99c72c5e5099e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - dnslib
+
+about:
+  home: "http://bitbucket.org/paulc/dnslib/"
+  license: "BSD"
```

Comment:
this should be bsd- 2-clause

Reply:
@scopatz fixed & test re-built; ready for review